pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample

```diff
@@ -128,6 +128,13 @@ export WIFIPASS='your_pass'
 # export WEBHOOK_ENABLED=true
 # export WEBHOOK_URL=http://domain/path
 
+# Uncomment if setting up Matrix notifications
+# export MATRIX_ENABLED=true
+# export MATRIX_SERVER_URL=https://matrix.org/
```

Comment:
you forgot to remove the trailing slash here

Reply:
Yes, thanks for catching that, trailing slash is now gone.